packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -22,6 +22,8 @@ brand-name-apple-app-store = App Store
 document =
   .title = Firefox Accounts
 
+brand-name-firefox-logo = { -brand-name-firefox } logo
```

Comment:
@bcolsson added as an `alt` for the Firefox logo that appears in the upper left corner of Subscriptions. I couldn't find an existing l10n id. Please let me know!